mesheryctl/internal/cli/root/model/build.go

```diff
@@ -0,0 +1,121 @@
+package model
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	meshkitOci "github.com/layer5io/meshkit/models/oci"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var buildModelCmd = &cobra.Command{
+	Use:   "build",
+	Short: "Create an OCI-compliant package from the model files",
+	Long: `Create an OCI-compliant package from the model files.
+Model files are taken from [path]/[model-name]/[version] folder.
+Expects input to be in the format scaffolded by the model init command.
+Documentation for exp model init can be found at https://docs.meshery.io/reference/mesheryctl/exp/model/init
+Documentation for exp model build can be found at https://docs.meshery.io/reference/mesheryctl/exp/model/build`,
```

Comment:
Couldn't we just point to one main link instead of sub links? Like below...

```suggestion
Documentation for exp model and subcommands can be found at https://docs.meshery.io/reference/mesheryctl/exp/model`,
```